### ISSUE TITLE
How to set parameters when use 940k ch-en corpus to train? 

### ISSUE THREAD

**Issue by Rooders** [THUNLP-MT/Document-Transformer#12]
Comparing with training of using corpus of 2M ch-en, when I use corpus of 940k ch-en  to train model, what parameters should I use ？I have tried to use batch_size=25k, learning rate=1.0 and batch_size=25k, learning rate=0.5,but just got bleu=3.6 and bleu=33.7 in MT06,respectively

**Comment by Glaceon31** [comment]
Do you initialize the document-level model with the sentence-level model? The default parameter setting in the code is OK. Just make sure you follow the steps in README
